### DISCUSSION TITLE
Hardware Encoder Question

### DISCUSSION THREAD

**Discussion by grndal** [cdgriffith/FastFlix#504] [Q&A] [answered]
I'm new to FastFlix and had a question about which encoders use the GPU vs the CPU.  I'm specifically interested in the AV1 encoder.  I'm running on Arc A770 and have found that some AV1 encoders use the GPU and some don't.  Is there anyway to know before kicking off an encode on what it will use?

Encoder
AV1 (SVT) - CPU
AV1F (SVT) - CPU
AV1 (QSVencC) - GPU

Hardware
i7-13700K
32GB DDR5
Arc A770 16GB

**Comment by cdgriffith** [comment, marked as answer]
Quick list while on mobile.

If you see an encoder that has one of these terms in it, it's for hardware

GPU (Hardware):
* QSV
* VCE
* AMF 
* NVENC
* OMX

**Comment by grndal** [comment]
Got it.  Thank you so much for the reply.
